### PR TITLE
Review

### PR DESCRIPTION
**Pull request by mllg**
Did some code reviews and cleanups.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+indent_size = 2
+
+[*.{c,h}]
+indent_size = 4
+
+[*.{cpp,hpp}]
+indent_size = 4
+
+[{NEWS.md,DESCRIPTION,LICENSE}]
+max_line_length = 80
```
Comment (RaphaelS1) [inline .editorconfig:21]: Not sure how much of a difference it makes but our lintr is set to 100 characters for length so maybe this should be too?

File: R/BenchmarkAggr.R [hunk 2]
```diff
@@ -8,15 +9,15 @@
...
+#' @references
+#' `r format_bib("demsar_2006")
 #'
 #' @examples
 #' # Not restricted to mlr3 objects
 #' df = data.frame(task_id = rep(c("A", "B"), each = 5),
 #'                 learner_id = paste0("L", 1:5),
 #'                 RMSE = runif(10), MAE = runif(10))
-#' BenchmarkAggr$new(df)
+#' as.BenchmarkAggr(df)
```
Comment (RaphaelS1) [inline R/BenchmarkAggr.R:20]: I had this as `BenchmarkAggr$new(df)` before just to highlight the main constructor, then have the same example for the coercion method later. Not sure how much of a difference it makes either way though

**Comment by RaphaelS1** [comment]
Thanks for this!

File: .editorconfig (hunk 1 above)
Comment (mllg) [reply .editorconfig:21]: IIRC, `usethis::use_tidy_description()` wraps at 80 chars.

**Pull request merged**